ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplUtil.java

```diff
@@ -99,8 +102,37 @@ public static void fillMapper(StringToIntMapper<AmplSubset> mapper, Network netw
 
         // HvdcConverterStations
         network.getHvdcConverterStations().forEach(conv ->
-                mapper.newInt(conv.getHvdcType().equals(HvdcType.VSC) ? AmplSubset.VSC_CONVERTER_STATION : AmplSubset.LCC_CONVERTER_STATION,  conv.getId()));
+                mapper.newInt(conv.getHvdcType().equals(HvdcType.VSC) ? AmplSubset.VSC_CONVERTER_STATION : AmplSubset.LCC_CONVERTER_STATION, conv.getId()));
+
+    }
+
+    public static Double getMinQ0(Generator gen) {
+        List<ReactiveCapabilityCurve.Point> point = getReactiveCapabilityCurvePoint(gen);
+        if (!point.isEmpty()) {
+            return point.get(0).getMinQ();
+        }
+        return AmplConstants.INVALID_DOUBLE_VALUE;
+    }
 
+    public static Double getMaxQ0(Generator gen) {
+        List<ReactiveCapabilityCurve.Point> point = getReactiveCapabilityCurvePoint(gen);
+        if (!point.isEmpty()) {
+            return point.get(0).getMaxQ();
+        }
+        return AmplConstants.INVALID_DOUBLE_VALUE;
+    }
+
+    private static List<ReactiveCapabilityCurve.Point> getReactiveCapabilityCurvePoint(Generator gen) {
+        ReactiveLimits reaLim = gen.getReactiveLimits();
+        List<ReactiveCapabilityCurve.Point> points = new ArrayList<>();
```

Comment:
You could check instanceof before this line. if true, return ... else return `Collections.emptyList()`